cluster/src/dunit/scala/io/snappydata/externalstore/ColumnTableDUnitTest.scala

```diff
@@ -33,33 +33,49 @@ import org.apache.spark.sql.{Row, SaveMode, SnappyContext}
  */
 class ColumnTableDUnitTest(s: String) extends ClusterManagerTestBase(s) {
 
-  val currenyLocatorPort = ClusterManagerTestBase.locPort
+  val currentLocatorPort = locPort
 
   def testTableCreation(): Unit = {
     startSparkJob()
   }
 
+  //After discussing with Namratam Commenting out this test as it causes other tests to fail
+/*
+
   def testTableCreationWithHA(): Unit = {
```

Comment:
This test fails intermittently .And if it fails all other subsequent Dunit test fails.I have raised a jira for this https://jira.snappydata.io/browse/SNAP-1006 .As per the stack trace it seems issue at store side.Sorry for the typo :)